utils/interfaces/_core.py

```diff
@@ -63,6 +63,7 @@ def ingest_file(self, src_path):
                 try:
                     data = json.load(f)
                 except json.decoder.JSONDecodeError:
+                    logger.exception(f"Error while decoding message {src_path}")
```

Comment:
The case where it's a transient error (partial write), seems to be a fairly infrequent error. The poller has some delay, so whenever this happens because of a partial write, it's usually done in next iteration.
AFAIK the next iteration is currently 1 second (lower with the next PR though).